.github/workflows/python-packages.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Python distributions to PyPI and TestPyPI v2
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Dont think we'll be pushing directly to main. Also, I don't think we should deploy packages to PyPl for every commit we push to a branch.